operators/Cargo.toml

```diff
@@ -50,11 +50,13 @@ tracing = "0.1"
 tracing-opentelemetry = "0.18"
 typetag = "0.2"
 uuid = { version = "1.1", features = ["serde", "v4", "v5"] }
+xgboost-rs = "0.1.2"
 
 [dev-dependencies]
 async-stream = "0.3"
 geo-rand = { git = "https://github.com/lelongg/geo-rand", tag = "v0.3.0" }
 rand = "0.8"
+ndarray = "0.15.6"
```

Comment:
only specify minor version, not patch

Reply:
https://github.com/geo-engine/geoengine/pull/639/commits/e6b35477617e9dd0f4484a89bd65ec08f388f378